app/views/forum/topic.scala

```diff
@@ -46,8 +46,8 @@ object topic:
             )
           ),
           p(
-            "Make sure to read ",
-            strong(a(href := routes.Page.loneBookmark("forum-etiquette"))("the forum etiquette"))
+            trans.read(),
+            strong(a(href := routes.Page.loneBookmark("forum-etiquette"))(trans.etiquette()))
```

Comment:
what you're doing here is called translation lego, and it won't work with most languages, that don't want to put the word "etiquette" at the end.

Look at how the lichess translations take arguments to solve this problem.